### PR TITLE
sort the Team list by name rather than id

### PR DESCRIPTION
**Pull request by slewis74**

### CODE REVIEW

File: public/tendersmash.js [hunk 1]
```diff
@@ -430,6 +430,11 @@ app.controller("mainController", function ($scope, $http, $sce, $q, $timeout, pr
     return promises;
   }
 
+  $scope.getOrderedLists = function() {
+    var sorted = _.sortBy($scope.lists, function(x) { return x.name; });
```
Comment (hnrkndrssn) [inline public/tendersmash.js]: You might want to do a `.toLowerCase()` on the name here so that the sort is case-insensitive

**Review by hnrkndrssn — CHANGES_REQUESTED** [review]
Other than case-insensitive sorting, LGTM 👍

File: public/tendersmash.js (hunk 1 above)
Comment (matt-richardson) [reply public/tendersmash.js]: The way it is right now works well, as all the `area/` ones come up after the names, which feels kinda nice. Not sure if they should be first or last though...

Comment (slewis74) [reply public/tendersmash.js]: yeah, I liked that they were together. area would be common, maybe custom sort so they always end up at the top would be nice?

**Review by hnrkndrssn — APPROVED** [review]
👍

**Pull request merged**